Contribute/content/provide-feedback.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Provide feedback for Microsoft Learn content
+description: Learn how to provide feedback to Microsoft Learn content teams using the new feedback experience for documentation and training content.
+author: carlyrevier
+ms.author: cahublou
+ms.topic: contributor-guide
+ms.prod: non-product-specific
+ms.custom: external-contributor-guide
+ms.date: 02/22/2024
+---
+
+# Provide feedback for Microsoft Learn content
+
+This article explains how to provide feedback for Microsoft Learn content using the new feedback experience. This unified feedback experience is now standard across all products that publish documentation or training content on Microsoft Learn. The improved system:
+
+- Simplifies our procedures to efficiently incorporate your feedback.
```

Comment:
the current list contains some that are subjective and may make some customers unhappy.  simplifies or enhances...and "respond" might be read as it get back to the customer which we for sure cannot even do anymore.  here is my suggested list..

- Allows for feedback in languages other than English.
- Standardizes the customer feedback experience across the Learn platform.
- Supports global accessibility for non-English customers.
- Removes the requirement of a git login for providing feedback.